src/components/pages/pages.component.js

```diff
@@ -80,7 +80,6 @@ class Pages extends React.Component {
     index = visiblePage.props.id || index;
 
     const additionalProps = {
-      className: visiblePage.props.className,
```

Comment:
shouldn't that remain here for backward compatibility?

Reply:
agree, I got carried away when stripping it clean.